### ISSUE TITLE
identify key fields of mysql

### ISSUE THREAD

**Issue by dkdbsd** [Netflix/metacat#260]
Hello. 
I wonder metacat can identify key fields of mysql or not.

I think key fields are important information,
but,  it looks like metacat does not show it.

**Comment by ajoymajumdar** [comment]
Yes, we do not identify key fields in mysql. We had a discussion around this where we should be showing the keys on a table. There are different types of keys. At the least we should mark columns that is a key or part of a composite key. We will add this feature. I will keep this issue open till then.
Thanks for raising the issue.
